Package.swift

```diff
@@ -1,34 +1,37 @@
 // swift-tools-version:5.3
-
 import PackageDescription
 
 let package = Package(
-    name: "TreeSitterSQL",
-    platforms: [.macOS(.v10_13), .iOS(.v11)],
+    name: "TreeSitterSql",
```

Comment:
Two meaningful changes to note:
- the Swift package name has been changed - I don't know if capitalization matters for this package manager
- the `exclude` paths have disappeared, but the effect on the package should be minimal.